mautic/mautic
# Keep the fingerprint hash for devices.

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Related user documentation PR URL | No
| Related developer documentation PR URL | No
| Issues addressed (#s or URLs) | No
| BC breaks? | No
| Deprecations? | No

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
We are sending the fingerprint hash with the tracking pixel request, while the value is not saved. The lead_devices table is a good place to keep this information, so that it can be easily used when needed.

#### Steps to test this PR:
1. Make a normal page request, so that a new page_hit record generated.
2. Make sure the fingerprint hash is added to the device record in the lead_devices table. Or the existing device record gets updated if it returning request.
